core/src/main/java/com/klaytn/caver/account/Account.java

```diff
@@ -0,0 +1,226 @@
+     */
+    public static Account createWithAccountKeyWeightedMultiSig(String address, String[] publicKeys) {
+        WeightedMultiSigOptions options = getDefaultOptions(publicKeys.length);
+        return createWithAccountKeyWeightedMultiSig(address, publicKeys, options);
+    }
+
+    /**
+     * Create an Account instance which has AccountKeyWeightedMultiSig as an accountKey
+     * @param address address of Account
+     * @param publicKeys List of public key array
+     * @param options List of WeightedMultiSigOptions
+     * @return Account
+     */
+    public static Account createWithAccountKeyWeightedMultiSig(String address, String[] publicKeys, WeightedMultiSigOptions options) {
+        if(options == null) {
+            throw new NullPointerException("The variable 'options' is undefined. To create an Account instance with AccountKeyWeightedMultiSig, 'options' should be defined.");
+        }
+        IAccountKey accountKey = AccountKeyWeightedMultiSig.fromPublicKeysAndOptions(publicKeys, options);
+        return new Account(address, accountKey);
+    }
+
+    /**
+     * Create an Account instance which has AccountKeyRoleBased as an accountKey
+     * This method set 1 to WeightedMultiSigOptions values(threshold, weights)
+     * @param address address of Account
+     * @param roleBasedPublicKey List of public key array
+     * @return Account
+     */
+    public static Account createWithAccountKeyRoleBased(String address, List<String[]> roleBasedPublicKey) {
+        List<WeightedMultiSigOptions> optionList = new ArrayList<>();
+
+        for(int i=0; i<roleBasedPublicKey.size(); i++) {
+            WeightedMultiSigOptions option = getDefaultOptions(roleBasedPublicKey.get(i).length);
+            optionList.add(option);
+        }
+
+        return createWithAccountKeyRoleBased(address, roleBasedPublicKey, optionList);
+    }
+
+    /**
+     * Create an Account instance which has AccountKeyRoleBased as an accountKey
+     * @param address address of Account
+     * @param roleBasedPublicKey List of public key array
+     * @param optionsList List of WeightedMultiSigOptions
+     * @return Account
+     */
+    public static Account createWithAccountKeyRoleBased(String address, List<String[]> roleBasedPublicKey, List<WeightedMultiSigOptions> optionsList) {
+        if (optionsList == null) {
+            throw new NullPointerException("The variable 'optionsList' is undefined. To create an Account instance with AccountKeyRoleBased, 'optionsList' should be defined");
+        }
+        IAccountKey accountKey = AccountKeyRoleBased.fromRoleBasedPublicKeysAndOptions(roleBasedPublicKey, optionsList);
+
+        return new Account(address, accountKey);
+    }
+
+    /**
+     * Set 1 to WeightedMultiSigOptions values(threshold, weights)
+     * @param arrayLength Length of public key array
+     * @return WeightedMultiSigOptions
+     */
+    private static WeightedMultiSigOptions getDefaultOptions(int arrayLength) {
```

Comment:
How about implement this fucntion to WeightedMultiSigOptions class?

Reply:
Okay. I added function in WeightedMultiSigOptions class